agent/tool-scripts/datalog/prometheus-metrics-datalog

```diff
@@ -5,15 +5,18 @@ import sys, json
 import os,subprocess,warnings,logging
 
 interval = int(sys.argv[2])
-master = sys.argv[1]
+host = sys.argv[1]
 tool_log = sys.argv[3]
 port = int(sys.argv[4])
 cert = sys.argv[5]
 key = sys.argv[6]
 while True:
     time_stamp = int(time.time())
     next_start = time_stamp + interval
-    command = "prom2json --cert=%s --key=%s https://%s:%d/metrics" % (cert, key, master, port)
+    if (cert == "None") and (key == "None"):
```

Comment:
What happens if cert is None and key is not? Or vice versa?

Reply:
Fixed the condition, if cert or key is not defined then the get requests are done without authentication.